packages/radio-group/src/vaadin-radio-button.js

```diff
@@ -151,6 +151,21 @@ class RadioButton extends SlotLabelMixin(
     return this.$.noop;
   }
 
+  /**
+   * Override __copyNodesToSlotTarget from SlotTargetMixin to show a warning.
+   * @override
+   * @protected
+   * @param {Array<Node>} nodes
```

Comment:
If every node is expected not to be `null`, then the exclamation mark is probably required.

```suggestion
   * @param {!Array<!Node>} nodes
```